tracer/src/Datadog.Trace/DiagnosticListeners/Endpoint.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="Endpoint.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.DiagnosticListeners
+{
+    /// <summary>
+    /// Endpoint for duck typing
+    /// </summary>
+    [DuckCopy]
+    internal struct Endpoint
```

Comment:
I don't think this is required any more after your updates?

Reply:
thanks! my bad I didnt see it 🤦

Reply:
fixed in 5b618b7